morphocut/stream.py

```diff
@@ -15,9 +15,18 @@
 
 class TQDM(Node):
     """
-    Provide a progress indicator via `tqdm`_.
+    Provide a progress indicator via `tqdm`_. Returns an iterator which prints a dynamically
+    updating progressbar.
 
-    .. _tqdm: https://github.com/tqdm/tqdm
+    ..Note:
+       An external dependency tqdm is required to use this
+
+       .. _tqdm: https://github.com/tqdm/tqdm
+
+    Example:
+       ..code-block:: python
+          for i in tqdm(range(10000)):
+             pass
```

Comment:
We don't need an example for tqdm, but on how to use `morphocut.stream.TQDM`. Obvious, isn't it?

Reply:
Also, this does not render properly when building the documentation.
https://trello.com/c/PyWg4rz3/13-build-the-documentation